Gopkg.lock

```diff
@@ -2,49 +2,59 @@
 
 
 [[projects]]
+  digest = "1:50e893a85575fa48dc4982a279e50e2fd8b74e4f7c587860c1e25c77083b8125"
```

Comment:
Does this file needs to be changed at all ?

Reply:
with the removal openvpn/core from go-openvpn the last mention of urfave/cli gets dropped, therefore it's been removed from the .lock file